website/static/js/forgotPassword.js

```diff
@@ -0,0 +1,54 @@
+/*
+ * forgot password view model
+ */
+'use strict';
+
+var ko = require('knockout');
+require('knockout-validation');
+
+var $osf = require('osfHelpers');
+var $formViewModel = require('formViewModel');
```

Comment:
Great question. I think I wanted to follow in line with other helper stuff but it looks weird.